smallest_multiple.js

```diff
@@ -8,7 +8,22 @@
  */
 module.exports = function( ceiling ) {
   // do work here
+  let numberToBeDivided = 1;
+  let smallestNumber;
+  let noRemainders = false;
+  
+  while (!noRemainders){
+    for (i = 1; i <= ceiling; i++){
+      if (numberToBeDivided % i !== 0){
+        break;
+      }
+      if (i === ceiling){
```

Comment:
This if conditional could be removed if you moved the check to see if the for loop found any numbers that didn't divide into `numberToBeDivided` outside of the for loop.